### DISCUSSION TITLE
AudioBookshelf LXC Unusable?

### DISCUSSION THREAD

**Discussion by HackerJL** [tteck/Proxmox#3634] [Help Needed] [answered]
Hey everyone,

Wondering if I am the only one that is unable to use AudioBookShelf using default settings?

Ive tried it on 2 different proxmox hosts, at different locations, and not been able to get it to function much.

Default install, works.  Local LAN, setup user - works good, can get to main page
Add a library (I called mine podcasts)
go in and add a podcast (I tried adding 'technado')
It fails wonderfully, and appears the audiobookshelf backend crashes.  

Like I said, did this on two different machines, so Im scratching my head...

Anyone else have this same issue?

![Screenshot 2024-08-31 at 6 19 28 PM](https://github.com/user-attachments/assets/6548a66e-5a95-43d9-9b01-c33987e73b6d)

**Comment by novajolt** [comment]
Oh im glad youve posted this. Ive been trying for past couple of hours and couldnt get it to work either. Just wont allow upload of podcasts or audiobooks!

**Reply by HackerJL** [reply]
Phew!  Its not just me.

**Comment by supperka** [comment]
Hello all my tries end with fail , I can´t upload any audiobooks.

**Comment by oOStroudyOo** [comment, marked as answer]
Looks like it might have been fixed with latest update: https://github.com/advplyr/audiobookshelf/releases/tag/v2.13.1

I believe the Tteck script just adds the source for audiobookshelf.

Running `apt-get update && apt-get upgrade` should fix your problem.

**Reply by supperka** [reply]
Hello , Im already have latest version. 2.13.1

after apt-get update && apt-get upgrade 

getting : The following packages will be upgraded:
  audiobookshelf
1 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Need to get 22.7 MB of archives.
After this operation, 0 B of additional disk space will be used.
E: You don't have enough free space in /var/cache/apt/archives/.

Im used defalut instalation

**Reply by tteck** [reply]
Increase your storage

![Screenshot 2024-09-01 5 13 18 PM](https://github.com/user-attachments/assets/805bc54d-34a8-4a3d-9481-71e057f434d3)

**Reply by tteck** [reply]
@oOStroudyOo is correct, Audiobookshelf is using their APT repository.
https://github.com/tteck/Proxmox/blob/e680dab28fd506e8ad0624275035c4fcb7ec48e8/install/audiobookshelf-install.sh#L23-L28

**Reply by oOStroudyOo** [reply]
v2.13.2 just got released 3 minutes ago.

Try another update once you give yourself some more storage space to do the update and then reboot the container.
Unless for some reason the app has lost access to the storage location, which I doubt considering the above was an actual issue with audiobookshelf, then I'd raise an issue on the audiobookshelf Github.

I highly doubt this error is anything that Tteck can fix consider how minimal the install script is for this.
It simply does just add it to your source code, and your container OS will do the install: https://github.com/tteck/Proxmox/blob/main/install/audiobookshelf-install.sh

**Reply by HackerJL** [reply]
I can confirm this worked for me, the OP.  I gave the container more hd space (from 4 to 14 - added 10), and did update.  Works nicely, now.
